src/cache-gradle-user-home.ts

```diff
@@ -13,25 +13,36 @@ const META_FILE = 'cache-metadata.json'
 
 const INCLUDE_PATHS_PARAMETER = 'gradle-home-cache-includes'
 const EXCLUDE_PATHS_PARAMETER = 'gradle-home-cache-excludes'
-const ARTIFACT_BUNDLES_PARAMETER = 'gradle-home-cache-artifact-bundles'
+const EXTRACTED_CACHE_ENTRIES_PARAMETER = 'gradle-home-extracted-cache-entries'
 
 /**
- * Represents the result of attempting to load or store a cache bundle entry.
+ * Represents the result of attempting to load or store an extracted cache entry.
```

Comment:
One thing that I think would help with maintainability here is some brief documentation around the format of cache-metadata.json.  I mean, you can infer it from `ExtractedCacheEntry`, but I think it would be easier to reason about with a quick example of different types of entries in the json file.  Not necessary as a part of this PR, just a suggestion for future improvements.

Reply:
Thanks. I'll think about how to explain this. The .json file is really just an implementation detail, and the pure serialized form of `ExtractedCacheEntry`. I've never actually considered what it looks like :).